libs/sdk/README.md

```diff
@@ -55,118 +49,113 @@ const sdk = getSDK();
+## Access Logging services
 
-Below is an example of how to create a logger associated with a specific app and then use it to log an error:
+Logging service is available under the log namespace inside services. Logging errors associated with a specific app is helpful when things go wrong because it can speed up the debugging process. Below is an example of how to create a logger associated with a specific app and then use it to log an error:
 
-```typescript
+```tsx
 import getSDK from "@akashaorg/core-sdk";
- getSDK().services.log.create('example-app-name');
 
-  if (error) {
-    logger.warn('Your error message here');
-    //other action
-  }
-```
+getSDK().services.log.create("example-app-name");
 
-#### The `api` namespace
+if (error) {
+  logger.warn("Your error message here");
+  // other action
+}
+```
 
-All API endpoints related to the authentication, profile, and global channel can be accessed via the `api` namespace.
+## API Namespace
 
-##### Authentication
+All API endpoints related to the authentication, profile, and global channel can be accessed via the API Namespace.
 
-Authentication-related endpoints are available under the `auth` namespace inside `api`.
+### Authentication
 
-For example, to get the current logged in user, you can use: 
+Authentication-related endpoints are available under the `Auth namespace` inside API. For example, to get the current logged in user, you can use:
 
-```typescript
+```tsx
 import getSDK from "@akashaorg/core-sdk";
+
 const currentUser = await getSDK().api.auth.getCurrentUser();
-// currentUser will be null when the user is not logged in, otherwise, it will be an object containing the DID and ethAddress (if any) of the logged in user. 
+// currentUser will be null when the user is not logged in,
+// otherwise, it will be an object containing the DID and ethAddress (if any) of the logged in user.
 ```
 
-##### Global channel
-
-Endpoints related to the SDK's global channel are available under the `globalChannel` namespace inside `api`.
+### Global Channel
 
-For example, to perform certain action when certain events are emitted, you can access the global channel and subscribe to those events:
+Endpoints related to the SDK's global channel are available under the `globalChannel namespace` inside API. For example, to perform certain action when certain events are emitted, you can access the global channel and subscribe to those events:
 
-```typescript
+```tsx
 import getSDK from "@akashaorg/core-sdk";
-  React.useEffect(() => {
-    const subSDK = sdk.api.globalChannel.subscribe({
-      next: (eventData: { data: { name: string }; event: APP_EVENTS }) => {
-        if (eventData.event === APP_EVENTS.INFO_READY && eventData.data.name === integrationName) {
-          // perform some actions here
-        }
-      },
-    });
-    return () => {
-      subSDK.unsubscribe();
-    };
-  }, []);
+React.useEffect(() => {
```

Comment:
can you refactor this example and remove react related apis? I know we are using react but the sdk can be used outside of react also.

Reply:
gotcha